src/languages/en.ts

```diff
@@ -2303,8 +2303,8 @@ export default {
         overLimitAttendee: ({formattedLimit}: ViolationsOverLimitParams) => `Amount over ${formattedLimit}/person limit`,
         perDayLimit: ({formattedLimit}: ViolationsPerDayLimitParams) => `Amount over daily ${formattedLimit}/person category limit`,
         receiptNotSmartScanned: 'Receipt not verified. Please confirm accuracy.',
-        receiptRequired: ({formattedLimit, category}: ViolationsReceiptRequiredParams = {}) =>
-            `Receipt required${formattedLimit ? ` over ${formattedLimit}${category ? ' category limit' : ''}` : ''}`,
+        receiptRequired: (params: ViolationsReceiptRequiredParams) => `Receipt required${params ? ` over ${params.formattedLimit}${params.category ? ` category limit` : ''}` : ''}`,
```

Comment:
Is there a reason for this change? I see we didn't make the same change in the es.ts file, can we undo this?

Reply:
We discussed it here: https://expensify.slack.com/archives/C01GTK53T8Q/p1704823942664409?thread_ts=1704302068.776969&cid=C01GTK53T8Q

I've updated them to be like this:

| Params | English Translation | Spanish Translation |
| --- | --- | --- |
| No params | `Receipt required` | `Recibo obligatorio` |
| `{ formattedLimit: '$500' }` | `Receipt required over $500` | `Recibo obligatorio para importes sobre $500` |
| `{ category: 'travel' }` | `Receipt required over category limit` | `Recibo obligatorio para importes sobre el límite de la categoría` |
| `{ formattedLimit: '$500', category: 'travel' }` | `Receipt required over $500 category limit` | `Recibo obligatorio para importes sobre el límite de la categoría $500` |

Reply:
@trevor-coleman @cead22  Coming from this issue https://github.com/Expensify/App/pull/44220 Can you confirm why we don't mention category name on the Spanish translation while we mention it on English translation? Are this intended? or we should fix it.
CC @abzokhattab

Reply:
We're not showing the category name in either case. We should say literally "category limit" or "límite de la categoría"

Reply:
> We should say literally "category limit" or "límite de la categoría"

@cead22 is this mean the above table is not correct? because it showing the category name in English translation

Reply:
I think so. To be honest, I don't remember noticing the "travel" in "travel category limit" in there

Reply:
The above line use `params` to check before showing word `over` which display `over undefined` if `params = {formattedLimit: undefined, category: undefined}` and we fixed it here https://github.com/Expensify/App/pull/44220